set_default.cpp

```diff
@@ -238,6 +238,10 @@ void set_default_input_values(void) {
   g_sequential_source_gamma_id_list[15] = _default_seq_source_gamma_id;
 
   g_sequential_source_gamma_id_number = _default_seq_source_gamma_id_number;
+  g_spintrace = _default_spintrace;
+  g_filtered_qsq = _default_filtered_qsq ;
+  g_currentgammas = _default_currentgammas ;
```

Comment:
so right now only a single spin projection is supported, correct?

Reply:
Yes, right now one has to run separately for each spin structures

Reply:
Do you think it would make sense to support multiple projections in one run? It would make the handling of the input files simpler... I still think it's good to produce multiple output files.